graphql/schema.graphql

```diff
@@ -22,13 +22,14 @@ type Mutation {
     items: [ItemInput]
     marketingData: MarketingDataInput
     salesChannel: String
-    allowOutdatedData: [String!]
+    allowedOutdatedData: [String!]
```

Comment:
isn't this a breaking change? is there any app using the `allowOutdatedData` argument?

Reply:
no one is using it yet, that's why I did not consider this a breaking change

Reply:
alright then